client/app/telecom/pack/slots/emailPro/detail/pack-emailPro-detail.html

```diff
@@ -0,0 +1,139 @@
+                              total: $ctrl.account.quota.value,
+                              totalUnit: ('email_pro_detail_'+$ctrl.account.quota.unit | translate)
+                        }"></dd>
+                    <!-- END OF QUOTA -->
+
+                    <!-- WEBMAIL -->
+                    <dt class="service-title" data-translate="email_pro_detail_webmail"></dt>
+                    <dd class="service-item">
+                        <a href="{{::$ctrl.configuration.webmailUrl}}"
+                           data-translate-attr="{title:'email_pro_detail_open_webmail'}"
+                           title="{{::$ctrl.configuration.webmailUrl}} {{::'common_newtab' | translate}}"
+                           rel="noopener"
+                           target="_blank">
+                            <span data-ng-bind="$ctrl.configuration.webmailUrl"></span>
+                            <span class="ovh-font ovh-font-newtab" aria-hidden="true"></span>
+                        </a>
+                    </dd> <!-- END OF WEBMAIL -->
+
+                    <!-- GUIDE -->
+                    <dt class="service-title" data-translate="email_pro_detail_need_help"></dt>
+                    <dd class="service-item">
+                        <a href="{{::$ctrl.guideUrl}}"
+                           data-translate-attr="{title:'email_pro_detail_guide'}"
+                           title="{{::$ctrl.guideUrl}} {{::'common_newtab' | translate}}"
+                           rel="noopener"
+                           target="_blank">
+                            <span data-translate="email_pro_detail_guide"></span>
+                            <span class="ovh-font ovh-font-newtab" aria-hidden="true"></span>
+                        </a>
+                    </dd> <!-- END OF GUIDE -->
+
+                </dl>
+            </div>
+        </div>
+
+        <div class="row"
+             data-ng-if="!$ctrl.loading">
+            <!-- Change password -->
+            <div class="col-md-6 col-xs-12 widget-presentation">
+                <h2 class="widget-presentation-title"
+                    data-translate="email_pro_detail_client_change_password">
+                </h2>
+                <form name=passwordChange data-ng-submit="$ctrl.changePassword()">
+                    <div
+                        data-ovh-password
+                        data-ovh-pwd="$ctrl.password"
+                        data-ovh-pwd-confirm="ctrl.account.passwordConfirmation">
+                    </div>
+
+                    <button class="btn btn-primary"
+                            type="submit"
+                            data-ng-disabled="passwordChange.$invalid || $ctrl.changingPassword">
+                            <span data-translate="email_pro_detail_client_change_password_submit"></span>
+                            <oui-spinner data-ng-if="$ctrl.changingPassword"
```

Comment:
See https://github.com/ovh-ux/ovh-manager-telecom/pull/756/files#diff-2fbd95b44257cc5fabd8ec6c43c704beR129.

You have a spinner and text inside your `<button>` element. Here is the opposite.